internal/cli/cmd/tenant/update.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright (c) 2024 OceanBase
+ob-operator is licensed under Mulan PSL v2.
+You can use this software according to the terms and conditions of the Mulan PSL v2.
+You may obtain a copy of Mulan PSL v2 at:
+
+	http://license.coscl.org.cn/MulanPSL2
+
+THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+See the Mulan PSL v2 for more details.
+*/
+package tenant
+
+import (
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/types"
+
+	cmdUtil "github.com/oceanbase/ob-operator/internal/cli/cmd/util"
+	"github.com/oceanbase/ob-operator/internal/cli/tenant"
+	"github.com/oceanbase/ob-operator/internal/clients"
+)
+
+// NewUpdateCmd update an obtenant
+func NewUpdateCmd() *cobra.Command {
+	o := tenant.NewUpdateOptions()
+	logger := cmdUtil.GetDefaultLoggerInstance()
+	cmd := &cobra.Command{
+		Use:     "update <tenant_name>",
+		Short:   "Update ob tenant",
+		Long:    "Update ob tenant, support unitNumber/charset/connectWhiteList/priority of zones",
```

Comment:
`charset` has already been removed from the valid operations. The description need to be updated.